cinema/migrations/0002_actor_cinemahall_genre_movie_actors_movie_genres.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.1 on 2024-12-06 14:32
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Actor',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=255)),
+                ('last_name', models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='CinemaHall',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('rows', models.PositiveIntegerField()),
+                ('seats_in_row', models.PositiveIntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Genre',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(related_name='actors', to='cinema.actor'),
+        ),
+        migrations.AddField(
+            model_name='movie',
+            name='genres',
+            field=models.ManyToManyField(related_name='genres', to='cinema.genre'),
```

Comment:
The `related_name` for the `genres` field is set to 'genres', which can cause confusion. Consider using a more descriptive name like 'movies' to better reflect the relationship from the other side.